src/scripts/settings/userSettings.js

```diff
@@ -107,7 +107,7 @@ import events from 'events';
         }
 
         val = this.get('enableThemeSongs', false);
-        return val !== 'false';
+        return val === 'true';
```

Comment:
These statements are not equivalent. Consider the case when`val` Is undefined or null.

Reply:
It was intentional, I was changing the default settings to something I consider more sane. I don't think we should enable the backdrops by default, but that one's up for discussion. Theme songs and videos are also rarely used so it doesn't make sense to enable them by default.

Reply:
Ah I see! I think backdrops should b enabled my default, but agree on the others.